scripts/sql/delta/v190-v200/apolloconfigdb-v190-v200.sql

```diff
@@ -37,6 +37,7 @@ ALTER TABLE `GrayReleaseRule`
 
 ALTER TABLE `Item`
     ADD COLUMN `DeletedAt` BIGINT(20) NOT NULL DEFAULT '0' COMMENT 'Delete timestamp based on milliseconds' AFTER `IsDeleted`,
+    ADD COLUMN `Type` tinyint(3) unsigned NOT NULL DEFAULT '0' COMMENT '配置项类型，0: String，1: Number，2: Boolean，3: Json' AFTER `Key`,
```

Comment:
apollo 2.0.0 is released, we need to add the upgrade scripts in a new folder - `v200-v210`.

Reply:
TKS. I have already move the ddl to the new file in a new folder - v200-v210